### ISSUE TITLE
After 'remove breakpoint' command, select next item in list instead of nothing/top

### ISSUE THREAD

**Issue by joerohde** [microsoft/vscode#96074]
When doing a 'remove breakpoint' from the 'Breakpoints' pane, interactively, or via cmd-backspace/debug.removeBreakpoint, the list UI selection is emptied, or left at the top.

PLEASE PLEASE select the item right after the breakpoint that was just deleted so it's reasonably easy to delete a series of breakpoints but not all of them.

Extra credit: It would be great if I could multi select that list and just do a 'remove breakpoint' command to apply all at once.

**Comment by isidorn** [comment]
This is a fair request.
We accept PRs that fix this.
Code pointer https://github.com/microsoft/vscode/blob/25e889af458c2be6b2ba2a3b5510b370586bb5c5/src/vs/workbench/contrib/debug/browser/breakpointsView.ts#L239

In case nobody tackles this in the near future I can also look into this.
Thanks!

**Comment by ChrisPapp** [comment]
@isidorn 
I created PR #96287. With this change, the cursor should now move to the next breakpoint after removing the focused breakpoint.

About multi-select, right now the _focused_ breakpoint is removed,
the one you see when you press up/down arrow keys,
not the _selected_,
the one you click on.

Therefore, if something like this were implemented, "Remove Breakpoint" should remove _selected_ breakpoints. But what would happen with the _focused_ breakpoint?
